packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -391,6 +392,22 @@ export abstract class SharedObjectCore<
 		localOpMetadata: unknown,
 	): void;
 
+	/* eslint-disable jsdoc/check-indentation */
+	/**
+	 * Derived classes should override this to do custom processing on remote messages.
+	 * Process a bunch of messages for this shared object. A bunch is group of messages that have the following properties:
+	 * - They are all part of the same grouped batch, which entails:
+	 *   - They are contiguous in sequencing order.
+	 *   - They are all from the same client.
+	 *   - They are all based on the same reference sequence number.
+	 *   - They are not interleaved with messages from other clients.
+	 * - They are not interleaved with messages from other DDS in the container.
```

Comment:
Missing "a"; quoting "bunch" so it's clearer it's not meant to be read in the usual sense; make it clear that derived classes should only override it _if they need to_. Some of this can apply to the docs for `private processMessages`.
```suggestion
	 * Process a 'bunch' of messages for this shared object.
	 *
	 * @remarks
	 * A 'bunch' is a group of messages that have the following properties:
	 * - They are all part of the same grouped batch, which entails:
	 *   - They are contiguous in sequencing order.
	 *   - They are all from the same client.
	 *   - They are all based on the same reference sequence number.
	 *   - They are not interleaved with messages from other clients.
	 * - They are not interleaved with messages from other DDS in the container.
	 * Derived classes should override this if they need to do custom processing on remote messages.
```